src/content/ja/2019/ecommerce.md

```diff
@@ -0,0 +1,608 @@
+</figure>
+
+ウェブサイトのデスクトップ比率は全体的に若干高くなっていますが、一部の人気プラットフォーム（WooCommerce、PrestaShop、Shopwareを含む）では、実際にはデスクトップウェブサイトよりもモバイル性が高くなっています。
+
+## ページの重さと要望
+
+Eコマースプラットフォームの[ページの重さ](./page-weight)は、すべてのHTML、CSS、JavaScript、JSON、XML、画像、オーディオ、およびビデオを含んでいます。
+
+<figure>
+  <a href="/static/images/2019/ecommerce/fig7.png">
+    <img src="/static/images/2019/ecommerce/fig7.png" alt="図7. Eコマースのページ重量の分布。" aria-labelledby="fig7-caption" aria-describedby="fig7-description" width="600" height="363" data-width="600" data-height="363" data-seamless data-frameborder="0" data-scrolling="no" data-iframe="https://docs.google.com/spreadsheets/d/e/2PACX-1vROHKGuMutXmxtzEoBSJNVn7DOzlfpizJh7mOkopFK8OVl_hCUHDOmKgYOojrpVsGnGWaletE7Uc5oX/pubchart?oid=448248428&amp;format=interactive">
+  </a>
+  <div id="fig7-description" class="visually-hidden">10、25、50、75、およびEコマースページの重量の90パーセンタイルの分布。中央値のデスクトップEコマースページは2.7MBをロードします。10パーセンタイルは1.0MB、25パーセンタイルは1.6MB、75パーセンタイルは4.5MB、90パーセンタイルは7.6MBです。デスクトップのウェブサイトは、メガバイトの10分の1の割合でモバイルよりもわずかに高くなっています。</div>
+  <figcaption id="fig7-caption">図7. Eコマースのページ重量の分布。</figcaption>
+</figure>
+
+<figure>
+  <a href="/static/images/2019/ecommerce/fig8.png">
+    <img src="/static/images/2019/ecommerce/fig8.png" alt="図8. Eコマースページごとのリクエストの分布。" aria-labelledby="fig8-caption" aria-describedby="fig8-description" width="600" height="363" data-width="600" data-height="363" data-seamless data-frameborder="0" data-scrolling="no" data-iframe="https://docs.google.com/spreadsheets/d/e/2PACX-1vROHKGuMutXmxtzEoBSJNVn7DOzlfpizJh7mOkopFK8OVl_hCUHDOmKgYOojrpVsGnGWaletE7Uc5oX/pubchart?oid=1968521689&amp;format=interactive">
+  </a>
+  <div id="fig8-description" class="visually-hidden">Eコマースページあたりのリクエストの10、25、50、75、および 90パーセンタイルの分布。中央値のデスクトップ Eコマースページは108リクエストを行います。10パーセンタイルは53リクエスト、25パーセンタイルは76、75パーセンタイルは153、90パーセンタイルは210です。デスクトップのウェブサイトは、約10リクエストでモバイルよりもわずかに高くなっています。</div>
+  <figcaption id="fig8-caption">図8. Eコマースページごとのリクエストの分布。</figcaption>
+</figure>
+
+デスクトップEコマースプラットフォームのページの読み込み量の中央値は108リクエストと2.7MBです。すべてのデスクトップページの重量の中央値は74リクエストと[1.9 MB](./page-weight#page-weight) です。言い換えれば、Eコマースページは他のウェブページよりも50％近く多くのリクエストを行い、ペイロードは約35％大きくなっています。比較すると、[amazon.com](https://amazon.com)のホームページは、最初のロード時に約5MBのページ重量に対して約300リクエストを行い、[ebay.com](https://ebay.com)は約3MBのページウェイトに対して約150リクエストを行います。Eコマースプラットフォーム上のホームページのページ重量とリクエスト数は、各パーセンタイルでモバイルの方が若干小さくなっていますが、すべてのEコマースのホームページの約10％が7MB以上をロードし200以上のリクエストをしています。
+
+このデータは、ホームページのペイロードとスクロールなしのリクエストを含んでいます。明らかに、最初のロードに必要なはずのファイル数よりも多くのファイルを取得しているように見えるサイトがかなりの割合で存在しています(中央値は100以上)。以下の[サードパーティのリクエストとバイト数](#third-party-requests-and-bytes)も参照してください。
+
+Eコマース・プラットフォーム上の多くのホームページが、なぜこれほど多くのリクエストを行い、これほど大きなペイロードを持つのかをよりよく理解するために、さらに調査を行う必要があります。著者らはEコマース・プラットフォーム上のホームページで、最初のロード時に数百回のリクエストを行い、数メガバイトのペイロードを持つホームページを定期的に目にします。リクエスト数とペイロードがパフォーマンスの問題であるならば、どのようにしてそれらを減らすことができるのでしょうか？
+
+## タイプ別のリクエストとペイロード
+
+以下の表は、デスクトップでのリクエストの場合のものです。
+
+<figure>
+  <table>
+    <thead>
+      <tr>
+        <th>Type</th>
```

Comment:
Should we translate type (aka filetype)?
```suggestion
        <th>ファイルの種類</th>
```

Reply:
Thanks, I fixed.